liota/core/package_manager.py

```diff
@@ -252,24 +252,22 @@ def __init__(self, name=None):
 
     def _cmd_handler_list(self, parameter):
         if parameter == "packages" or parameter == "pkg":
-            log.warning("List of packages - \n\t%s"
-                        % "\n\t".join(sorted(
-                            self._packages_loaded.keys()
-                        ))
-                        )
+            loaded_list = sorted(self._packages_loaded.keys())
+            log.warning("List of packages - \t%s" % "\t".join(loaded_list))
+            print "loaded packages:", ",".join(loaded_list)
```

Comment:
I have discussed with Arun in email threads. Our package manager uses a loop thread to keep reading commands from command message queue, and could not return. Therefore, I could not pass a command execution status code back. Arun agrees that to get status from stdout, and in fact, he just wanted to know the execution status.

Reply:
What about the non-pulse-agent users? I don't think it is a good idea to send output to stdout, in a non-blocking format. It will just pop out on the shell, when they least expect.

Ideally, there should be a two-way channel, not a unidirectional pipe between liotapkg and liotad.

Reply:
Also, if package_manager is printing these values to stdout, how will these values reach the Pulse Agent? Pulse Agent might be expecting it at the time of calling liotapkg.sh, correct?